Gists/ScoreAfterRetrieveAnomaly.cs

```diff
@@ -0,0 +1,36 @@
+﻿using BigML;
+using System;
+using System.Threading.Tasks;
+
+namespace Demo
+{
+    class ScoreAfterRetrieveAnomaly
+    {
+        static async void Main()
+        {
+            // New BigML client with username and API key
+            Console.Write("user: "); var User = Console.ReadLine();
+            Console.Write("key: "); var ApiKey = Console.ReadLine();
+            var client = new Client(User, ApiKey);
+
+            // retrieve a anomaly detector with a known ID
+            Anomaly anomaly;
+            string anomalyId = "anomaly/54daa82eaf447f5daa000XXY"; //Put your ID here
+            while ((anomaly = await client.Get<Anomaly>(anomalyId)).StatusMessage.StatusCode != Code.Finished) await Task.Delay(10);
```

Comment:
El 15/07/2015 a las 11:10, Oscar Rovira escribió:
>
> In Gists/ScoreAfterRetrieveAnomaly.cs 
> <https://github.com/bigmlcom/bigml-csharp/pull/24#discussion_r34659550>:
>
> > +
> > +namespace Demo
> > +{
> > +    class ScoreAfterRetrieveAnomaly
> > +    {
> > +        static async void Main()
> > +        {
> > +            // New BigML client with username and API key
> > +            Console.Write("user: "); var User = Console.ReadLine();
> > +            Console.Write("key: "); var ApiKey = Console.ReadLine();
> > +            var client = new Client(User, ApiKey);
> > +
> > +            // retrieve a anomaly detector with a known ID
> > +            Anomaly anomaly;
> > +            string anomalyId = "anomaly/54daa82eaf447f5daa000XXY"; //Put your ID here
> > +            while ((anomaly = await client.Get<Anomaly>(anomalyId)).StatusMessage.StatusCode != Code.Finished) await Task.Delay(10);
>
> GET is a synchronous request, do we really need a long polling process 
> here?
>

No. Just copied & pasted code from other API request. Otherhand status 
checking should be kept.



> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bigmlcom/bigml-csharp/pull/24/files#r34659550>.
>